content/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md

```diff
@@ -0,0 +1,954 @@
+      -->
+      
+      serving 和 ready 非常相似。唯一的不同在于,
+      即便某端点的状态为 Terminating 也可以设置 serving。
+      对于处在终止过程中的就绪端点，此状况应被设置为 “true”。
+      如果设置为 nil，则消费者应该以 ready 值为准。
+      可以在 EndpointSliceTerminatingCondition 特性开关中启用此字段。
+
+    - **endpoints.conditions.terminating** (boolean)
+      
+      <!--
+      terminating indicates that this endpoint is terminating. A nil value indicates an unknown state. Consumers should interpret this unknown state to mean that the endpoint is not terminating. This field can be enabled with the EndpointSliceTerminatingCondition feature gate.
+      -->
+      
+      terminating 说明当前端点正在终止过程中。
+      nil 值表示状态未知。
+      消费者应将这种未知状态视为端点并不处于终止过程中。
+      可以通过 EndpointSliceTerminatingCondition 特性门控中开启此字段。
+
+  - **endpoints.deprecatedTopology** (map[string]string)
+    
+    <!--
+    deprecatedTopology contains topology information part of the v1beta1 API. This field is deprecated, and will be removed when the v1beta1 API is removed (no sooner than kubernetes v1.24). While this field can hold values, it is not writable through the v1 API, and any attempts to write to it will be silently ignored. Topology information can be found in the zone and nodeName fields instead.
+    -->
+    
+    deprecatedTopology 包含 v1beta1 API 的拓扑信息部分。目前已经弃用了此字段，
+    移除 v1beta1 API 时（不早于 Kubernetes v1.24）会一起移除此字段。
+    此字段目前仍然可以存储值，但是不能通过 v1 API 写入数据。
+    向此字段写入数据的任何尝试都会被忽略，并且不会通知用户。
+    移除此字段后，可以在 zone 和 nodeName 字段中查看拓扑信息。
+
+  - **endpoints.hints** (EndpointHints)
+
+    <!--
+    hints contains information associated with how an endpoint should be consumed.
+    -->
+    
+    hints 是关于应该如何使用某端点的提示信息。
+
+    <a name="EndpointHints"></a>
+
+    <!--
+    *EndpointHints provides hints describing how an endpoint should be consumed.*
+    -->
+    
+    **EndpointHints 提供应该如何使用某端点的提示信息。**
+
+    - **endpoints.hints.forZones** ([]ForZone)
+
+      <!--
+      *Atomic: will be replaced during a merge*
+      -->
+      
+      **原子性：合并期间将被替换**
+
+      <!--
+      forZones indicates the zone(s) this endpoint should be consumed by to enable topology aware routing.
+      -->
+      
+      forZones 表示应该由哪个可用区调用此端点从才能激活拓扑感知路由。
+
+      <a name="ForZone"></a>
+
+      <!--
+      *ForZone provides information about which zones should consume this endpoint.*
+      -->
+      
+      **ForZone 指示应该由哪些可用区调度此端点。**
+
+      - **endpoints.hints.forZones.name** (string), <!--required-->必需
+
+        <!--
+        name represents the name of the zone.
+        -->
+        
+        name 代表可用区的名称。
+
+  - **endpoints.hostname** (string)
+
+    <!--
+    hostname of this endpoint. This field may be used by consumers of endpoints to distinguish endpoints from each other (e.g. in DNS names). Multiple endpoints which use the same hostname should be considered fungible (e.g. multiple A values in DNS). Must be lowercase and pass DNS Label (RFC 1123) validation.
+    -->
+    
+    此端点的主机名称。端点的使用者可以通过此字段区分各个

```

Comment:
You only can mark a comment solved when you totally accept the suggestion. If reject or partial acceptable, give a feedback.